client/my-sites/checkout/composite-checkout/components/wp-checkout-order-review.tsx

```diff
@@ -133,6 +159,7 @@ export default function WPCheckoutOrderReview( {
 	};
 
 	const planIsP2Plus = hasP2PlusPlan( responseCart );
+	const shouldShowDomainNote = experiment?.variationName && domainUrl?.includes( 'wordpress.com' );
```

Comment:
Noting that `.includes( 'wordpress.com' )` will be positive also for domains like:

* `maliciouswordpress.com`
* `wordpress.com.somemalicioussite.com`
* `wordpress.comalicioussite.xyz`

Should we be checking if the domain ends on `.wordpress.com` instead?

Reply:
Good point @tyxla, however I don't think it's possible to map or transfer a domain that contains "wordpress" as it's a blacklisted string, and this is checked in the domain suggestions API 2b0a4-pb.

Reply:
Glad to hear! I guess my comment was mostly about precision here. Having a precedent for an imprecise check somewhere can easily lead to introducing more imprecision in the future, and it could happen in a place where we don't perform a special verification.

Reply:
Yes indeed, and it makes sense to be specific in code to avoid unintended consequences down the line. Since this is an experiment, I don't foresee an immediate issue, but if the variant wins and we decide to launch it, we should definitely make the change you suggested. Appreciate your eyes on this @tyxla , thanks!